tests/e2e/config/bootstrap.js

```diff
@@ -66,6 +64,116 @@ const OBSERVED_CONSOLE_MESSAGE_TYPES = {
 	info: 'log',
 };
 
+/**
+ * Array of console messages to ignore.
+ */
+const IGNORE_CONSOLE_MESSAGES = [
+	// An exception is made for _blanket_ deprecation warnings: Those
+	// which log regardless of whether a deprecated feature is in use.
+	{
+		matcher: 'includes',
+		pattern: 'This is a global warning',
+	},
+	// Viewing posts on the front end can result in this error, which
+	// has nothing to do with Gutenberg.
+	{
+		matcher: 'includes',
+		pattern: 'net::ERR_UNKNOWN_URL_SCHEME',
+	},
+	// We log this separately now in a way which includes the URL
+	// which is much more useful than this message.
+	{
+		matcher: 'startsWith',
+		pattern:
+			'Failed to load resource: the server responded with a status of',
+	},
+	// WordPress font loading warnings
```

Comment:
Comments should end in a full stop. I'll apply this change myself to save the round trip.

```suggestion
	// WordPress font loading warnings.
```